### ISSUE TITLE
Pixellating and picture that freezes

### ISSUE THREAD

**Issue by darfielddx** [Ascoware/get-iplayer-automator#176]
I upgraded to 1.13.4 and have had great problems since. The picture pixellates and freezes while to audio carries on. 

I am using OSX High Sierra 10.13.4 wiht a VPN. 

There are numerous programmes I have been trying to download and all have the same problem. It has been going on since I downloaded the latest version. Such an example is the film Finding Neverland, but there are many more. Gardener's World last week is another example.

I have tried removing the programme and reloading an older version, but without a lot of success.

**Comment by skovatch** [comment]
I think we'll need more information. What are you BBC TV settings? 50 FPS or not, etc.?

**Comment by darfielddx** [comment]
I am attaching a couple of screenshots. No, I don’t use 50 fps. I tried that and got a horrible download that jerked all over the place. 

The downloaded programmes are pixelated and tend to freeze while the sound carries on. 

I am about to go out, but when I get home I will find such a programme, put it in a folder and send you the link so you can see what I am talking about. On one of my computers I have gone back to 1.12 and it seems better. They system has been working perfectly until recently. I have used it now for several years and have been delighted with it - until recently.

Cheers

Mark


Mark and Maureen Forsyth

36 Summer Glades
Jalan Sri Tasik Barat 1B
63000 Cyberjaya
Selangor
MALAYSIA

**Comment by mutantcamel** [comment]
I would remove your name and address from posts given the "grey area" that we're operating in with this app.

**Comment by darfielddx** [comment]
Oops - yes. I saw it too late

**Comment by skovatch** [comment]
I don't think the link made it. Try sending it to 329836+skovatch@users.noreply.github.com  and hopefully I'll get notified that way.

**Comment by darfielddx** [comment]
Scott, did you get the link to the tv prog??

The pixellation and freezing of the film while the audio carries on is driving me mad!

**Comment by skovatch** [comment]
No, I never got anything. I hesitate to do this, but send the link to skovatch@gmail.com -- this part of the message will self-destruct in 24 hours. :-)

**Comment by darfielddx** [comment]
I’m at work, so I will do it when i get home in about 3 hours time. All
three of the computers I use have the same problem.

All 3 are using OS 10.13.4 - I have attached a screenshot of my work
computer - it is virtually identical to my home computers. We are all using
the same OS version.

I am attaching soem screenshots. Again, the settings are duplicated on my
other two computers.

Any help would be very much appreciated. I am fed up with looking at
pixellated programs!

Cheers and thanks, Mark

**Comment by darfielddx** [comment]
These are the shots from my main computer -as you will see they are pretty much the same.
